docquality.rst

```diff
@@ -91,4 +86,10 @@ installed.  The devguide HTML can be built by running::
 in the checkout directory, which will write the files to the ``_build/html``
 directory.
 
+Changes to the devguide are normally published within a day, on a schedule
+that may be different from the main documentation.
+
+.. _separate repository: https://github.com/python/devguide
+.. _devguide repo: https://github.com/python/devguide
```

Comment:
Hi @ezio-melotti,

You can chain the links as follows:
```
_separate repository:
_devguide_repo: https://github.com/python/devguide
```

Both labels will direct to the same link.